### PR TITLE
Fix ssh connection problem and wrong call get_disk_source issue

### PR DESCRIPTION
**Pull request by xiaodwan**
1. Fix hanging up in v2v ssh connection
2. Fix cannot find function get_disk_source in vm_xml

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.vmx.vmx_ssh.latest8.output_mode.rhev.rhv_upload: PASS (2073.85 s)

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.vmx.vmx_ssh.latest7.output_mode.rhev.rhv_upload: PASS (1953.41 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh please help to review this PR, thanks.

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -1179,7 +1179,9 @@ def setup_esx_ssh_key(hostname, username, password, port=22):
             port=port,
             password=password,
             prompt=r"[\#\$\[\]]",
-            preferred_authenticaton='password,keyboard-interactive')
+            verbose=True,
+            preferred_authenticaton='password,keyboard-interactive',
+            user_known_hosts_file='~/.ssh/known_hosts')
```
Comment (chunfuwen) [inline virttest/utils_v2v.py]: $HOME/.ssh/known_hosts maybe more safer in some situation than '~/.ssh/known_hosts' when assign value to user_known_hosts_file

**Comment by xiaodwan** [comment]
The manual of ssh_config says 'The default is ~/.ssh/known_hosts,
~/.ssh/known_hosts2.'.
And I think It's fine.

File: virttest/utils_v2v.py (hunk 1 above)
Comment (xiaodwan) [reply virttest/utils_v2v.py]: Done

**Pull request merged**